docker/images/interactive/build.sh

```diff
@@ -8,12 +8,17 @@ set -o nounset   # abort on unbound variable
 set -o pipefail  # don't hide errors within pipes
 
 readonly image_repository='3rdman'
-readonly supported_apache_spark_versions=("2.3.3" "2.3.4" "2.4.0" "2.4.1" "2.4.3" "2.4.4" "2.4.5" "2.4.6")
-readonly supported_dotnet_spark_versions=("0.9.0" "0.10.0" "0.11.0" "0.12.1")
+readonly supported_apache_spark_versions=(
```

Comment:
This may be a question for Spark team.  Thoughts on how to keep this version list up-to-date and other versions included in this script up-to-date?  It feels like there should be long term plans for getting this updated "automatically" as part of the release process.  Without this they will become stale and/or be a maintenance burden.

Reply:
Agreed